views/update/confirm-overseas-entity-details.html

```diff
@@ -0,0 +1,87 @@
+{% extends "layout.html" %}
+
+{% set title = "Confirm Overseas Data" %}
+
+{% block pageTitle %}
+  {% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    {% include "includes/list/errors.html" %}
+    <h1 class="govuk-heading-xl">Confirm this is the registered overeseas entity (NOT LIVE)</h1>
+
+  {% from "govuk/components/table/macro.njk" import govukTable %}
+  {{ govukTable({
+    captionClasses: "govuk-table__caption--m",
+    firstCellIsHeader: true,
+    rows: [
+      [
+        {
+          text: "Overseas Entity Name",
+          attributes: {
+            "id": "oe_entity_name"
```

Comment:
Has this file not been updated? I'd expect appData.entity.name or something here?